libs/core/src/components/pds-copytext/test/pds-copytext.spec.tsx

```diff
@@ -0,0 +1,149 @@
+import { newSpecPage } from '@stencil/core/testing';
```

Comment:
> **Note**

Coverage is currently less than desired, but this is due to a limitation with jest and the `document.execCommand` used in the fallback code. Everything else has coverage except for the safari fallback but the fallback is covered in the e2e test.